guides/v2.0/contributor-guide/contributing.md

```diff
@@ -214,10 +214,15 @@ Refer to the following table for a description of each label. These labels are a
       <tr>
          <td><img src="{{ site.baseurl }}common/images/github_Cannot_Reproduce.png" alt="Can Not Reproduce"/></td>
          <td>The Community Engineering Team can not reproduced the issue following steps to reproduce.</td>
-      </tr>  
+      </tr>
       <tr>
          <td><img src="{{ site.baseurl }}common/images/github_non-issue.png" alt="Non Issue"/></td>
          <td>A described behavior in the issue description is valid and shouldn't be changed in Magento code base.</td>
       </tr>
    </tbody>
+
```

Comment:
This section needs to be moved above the _Labels applied by the Magento team_ heading (line 123).